kerchunk/kerchunk-in-practice.ipynb

```diff
@@ -0,0 +1,2731 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        f.write(ujson.dumps(multi_kerchunk).encode())
</pre><p>It might be simpler to only use <code>json</code> and not <code>ujson</code> , which would also remove a dependency?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>